### ISSUE TITLE
Ignore pinch gestures on touch devices?

### ISSUE THREAD

**Issue by frankharrison-wj** [smashingboxes/OwlCarousel2#1237]
Is there any way to have Owl Carousel ignore pinch gestures on touch devices? I'm using PanZoom (https://github.com/timmywil/jquery.panzoom) on each image within my carousel, so you can swipe left and right to move through images (with Owl), and pinch to zoom each image in and out (with PanZoom). But sometimes when you're trying to zoom and image, Owl picks up the pinch gesture and moves back/forwards through the carousel.

Amazing plugin btw – used it a lot now and loved it!

**Comment by emredurukn** [comment]
Hi @frankharrison-wj is there any progress in this issue.

**Comment by frankharrison-wj** [comment]
@emredurukn no, don't think so. I think I just decided to live with it as it was...

**Comment by kennnielsen** [comment]
@frankharrison-wj 
Hi Frank,

Just curious. Now it has been a few years since the last comment. Was any solution ever found for Owl Carousel 2? 
Still having the issue using a theme having this included in the product gallery. 

Cheers!

**Comment by DegrizNet** [comment]
This is probably not used but maybe somebody still uses it.. this is fix..
`	Owl.prototype.onDragStart = function(event) {
        if (event.type === 'touchstart' && event.originalEvent.touches.length > 1) {
            return;
        }`
